pkgs/by-name/vb/vboot-utils/package.nix

```
@@ -3,36 +3,62 @@
   stdenv,
   fetchFromGitiles,
   pkg-config,
+  makeWrapper,
   libuuid,
   openssl,
   libyaml,
+  nss,
+  flashrom,
   xz,
 }:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "vboot-utils";
+  version = "140.16371";
 
-stdenv.mkDerivation {
-  version = "111.15329";
-
-  pname = "vboot_reference";
+  versionFormatted = lib.concatStringsSep "-" (lib.versions.splitVersion finalAttrs.version);
+  rev = "refs/heads/release-R${finalAttrs.versionFormatted}.B";
 
   src = fetchFromGitiles {
+    inherit (finalAttrs) rev;
     url = "https://chromium.googlesource.com/chromiumos/platform/vboot_reference";
```

```suggestion
    url = "https://chromium.googlesource.com/chromiumos/platform/vboot_reference";
    rev = "refs/heads/release-R${finalAttrs.versionFormatted}.B";
```
This should be accesible via src.rev